Bottom: 0a993b41e6c7690805dec5066e794f313d4ebab0
Top:    bbb2e472a7c58ce477a7bf050297258398ed4bba
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ae4ebd2..e7dff21 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1219,8 +1219,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                               copper_appearance,
                               NULL));
-
-    //break;
   }
